crates/wasm/src/imports.rs

```diff
@@ -36,7 +36,12 @@ extern "C" {
     // return value of 0 means ok; any value other than 0 means error.
     fn secp256r1_verify(msg_hash_ptr: usize, sig_ptr: usize, pk_ptr: usize) -> i32;
     fn secp256k1_verify(msg_hash_ptr: usize, sig_ptr: usize, pk_ptr: usize) -> i32;
-    fn secp256k1_pubkey_recover(msg_hash_ptr: usize, sig_ptr: usize, recovery_id: u8) -> usize;
+    fn secp256k1_pubkey_recover(
+        msg_hash_ptr: usize,
+        sig_ptr: usize,
+        recovery_id: u8,
+        compressed: u8,
```

Comment:
Using a `bool` for `compressed` doesn't work?

Reply:
bool shouldn't be supported in export "C"